### PR TITLE
Reverse Engineering Update Pr Port/ Board Cost Balance 

### PR DESCRIPTION
**Pull request by Squishy77**
Ports the re update from delta and adds the Rcd and its ammo to the re machine along with jaws of life for more potential usage. Also moves the Re components to the Bottom of each yml item for items with the Re component placed in the middle likely from time and neglect, so it's consistent and less messy and better to understand. Also makes the cost of the naynoboards consistent with the other boards 

Expands the re machines catalog and adds it to alternate research tech as well as making it craft able via plasma glass, makes all Nanyo-boards the same cost as the ones from the hyper lathe updates such 
I think this would help those who wish to work on the re machine helps with future updates I need to flesh out too such as the other syndicate suits and items, adds a little more use and makes it consistent with another tech in nt's catalog. The Remechine has been neglected for a long time, and it seems that super parts are in the work here so ill likely make the suits and wait for the updates rather than making a super re-machine like planed.
--
Media 

Screenshot 2024-09-11 114101
Screenshot 2024-09-11 114140
Screenshot 2024-09-11 113137

---

# Changelog

:cl:
- add: added rcds to the list of possible re and ported over the re update from delta 
- tweak: tweaked the costs of boards re-machine boards to be more consistent with their post hyper lathe update counter parts

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]

**Comment by VMSolidus** [comment]
image

Test fails are legitimate

**Comment by Squishy77** [comment]
damn ill get fixing

**Review by OldDanceJacket — APPROVED** [review]

File: Resources/Prototypes/Nyanotrasen/Recipes/Lathes/electronics.yml [hunk 1]
```diff
@@ -1,28 +1,28 @@
-- type: latheRecipe
-  id: ReverseEngineeringMachineCircuitboard
-  result: ReverseEngineeringMachineCircuitboard
-  completetime: 4
-  materials:
-     Steel: 100
-     Glass: 900
-     Gold: 100
```
Comment (VMSolidus) [inline Resources/Prototypes/Nyanotrasen/Recipes/Lathes/electronics.yml:8]: Latest error is being directly caused by commenting these out.

**Comment by Squishy77** [comment]
damn thought that would fix it

File: Resources/Prototypes/Nyanotrasen/Recipes/Lathes/electronics.yml [hunk 2]
```diff
@@ -13,16 +13,16 @@
...
-  completetime: 4
-  materials:
-     Steel: 100
-     Glass: 900
+ id: CrewMonitoringComputerCircuitboard
+ result: CrewMonitoringComputerCircuitboard
+ completetime: 4
+ materials:
+    Steel: 100
+    Glass: 400
```
Comment (VMSolidus) [inline Resources/Prototypes/Nyanotrasen/Recipes/Lathes/electronics.yml:25]:
```suggestion
   id: CrewMonitoringComputerCircuitboard
   result: CrewMonitoringComputerCircuitboard
   completetime: 4
   materials:
      Steel: 100
      Glass: 400
```

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**